hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/OzoneBucket.java

```diff
@@ -144,6 +150,22 @@ public OzoneBucket(ConfigurationSource conf, ClientProtocol proxy,
     this.creationTime = Instant.ofEpochMilli(creationTime);
     this.metadata = metadata;
     this.encryptionKeyName = encryptionKeyName;
+    long modifiedTime = Time.now();
```

Comment:
This can be simplified using Instant without conversions.
 
this.modifiedTime = Instant.now();
if (modifiedTime.isBefore(this.creationTime)) {
  this.modificationTime = this.creationTime.clone();
}